wwwapp/models.py

```diff
@@ -208,15 +210,29 @@ def __str__(self):
         return '%d: %s' % (self.year, self.name)
 
 
+class WorkshopStatus(Enum):
```

Comment:
Tbh I prefer @cytadela8's solution with constants - Django operates on strings anyway, so if you use an Enum you have to type `.name` every single time you want to use it...

Reply:
You don't need to write ```.name``` everywhere, it's here just for backwards compatibility before someone refactors the enums properly.

Reply:
Still, in this form it looks ugly, introduces even more mess to this code (we are already in a situation where one class has proper constants and the other does not) and nobody even asked for this change in this PR. Do it properly in a separate one if you have to.